### DISCUSSION TITLE
Winget Package Manifest Addition

### DISCUSSION THREAD

**Discussion by mgaryGCM** [Romanitho/Winget-AutoUpdate#292] [Ideas]
Are there any thoughts to adding this to the winget package manifest? This would enable someone to install it with a command such as "winget install winget-wau". This would greatly simplify a lot of the Intune-based installation issues we and many others are facing.

Someone has already attempted this via a 3rd party MS Store wrapper linked below. My trust is lower when using a wrapper for a wrapper via another 3rd party though.
https://apps.microsoft.com/store/detail/wingetautoupdateconfigurator/XP89BSK82W9J28

**Comment by KnifMelti** [comment]
Hi!
That someone is Hauke ([Weatherlights](https://github.com/Weatherlights/Winget-AutoUpdate-Intune)) with a fork of this Repo and a MSI in Microsoft Store.
A winget package has been proposed in #60 and especially in #215
https://github.com/Romanitho/Winget-AutoUpdate/issues/215#issuecomment-1291202543 says the most

**Reply by KnifMelti** [reply]
Could you elaborate on the "_**../a lot of the Intune-based installation issues/..**_"
